TDD/TDD1_BerlinClock/src/main/java/logic/impl/BerlinClock.java

```diff
@@ -0,0 +1,96 @@
+package logic.impl;
+
+import logic.Clock;
+
+public class BerlinClock implements Clock {
```

Comment:
As a consequence of poorly defined interface, this class has only public methods.
That's not bad in all cases, but it's wrong in most cases.
Please, rethink whole application.